include/core/CBoostJsonPoolAllocator.h

```diff
@@ -36,46 +36,40 @@ namespace core {
 //!
 class CBoostJsonPoolAllocator {
 public:
-    using TDocumentWeakPtr = std::weak_ptr<boost::json::object>;
-    using TDocumentPtr = std::shared_ptr<boost::json::object>;
+    using TDocumentWeakPtr = std::weak_ptr<json::object>;
+    using TDocumentPtr = std::shared_ptr<json::object>;
     using TDocumentPtrVec = std::vector<TDocumentPtr>;
 
 public:
-    CBoostJsonPoolAllocator() : m_JsonPoolAllocator(m_FixedBuffer) {}
+    CBoostJsonPoolAllocator() {}
 
-    ~CBoostJsonPoolAllocator() { this->clear(); }
-
-    void clear() { m_JsonPoolAllocator.release(); }
+    ~CBoostJsonPoolAllocator() {}
```

Comment:
There's no need to declare an empty, non-virtual, destructor. The default one is fine.